torch/csrc/api/include/torch/data/worker_exception.h

```diff
@@ -13,7 +13,6 @@ namespace data {
 struct WorkerException : public std::exception {
   /// Constructs a `WorkerException` from an `exception_ptr`.
   explicit WorkerException(std::exception_ptr original)
-      // NOLINTNEXTLINE(performance-move-const-arg)
```

Comment:
The standard doesn't actually guarantee std::exception_ptr has a move-ctor, so I could see how it could complain about it.

Reply:
It depends on implementation. But cppreference says that "std::exception_ptr is a shared-ownership smart pointer'', so it is not harm to move it.